spec/models/medicaid_application_member_attributes_spec.rb

```diff
@@ -60,6 +69,13 @@
         primary_member_second_employed_employer_name: "BB Burgers",
         primary_member_second_employed_pay_interval_weekly: "Yes",
         primary_member_second_employed_pay_quantity: "222",
+        primary_member_additional_income_none: nil,
```

Comment:
this seems unrelated to this PR?

Reply:
Agree, there are a lot of different things happening. Hard to sift through what's what.